syntax = "proto3";

import "protocol/protocol_messages.proto";



service HelloService {
  rpc SayHello (HelloRequest) returns (HelloResponse);
}

message HelloRequest {
  InfosHeader info = 1 ;
  SetStateArgs set_state_args = 2;
}

message HelloResponse {
  string message = 1;
}


message SetStateArgs {
  int32 io_address = 1;
  int32 new_state = 2; 
}